### DISCUSSION TITLE
how to use extendManifestJson (json) and process.env vars in Quasar2/vite

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#13857] [CLI - PWA mode] [answered]
In the old webpack world, I could do something like this in the pwa block of my quasar.conf.js file:
```
pwa: {
      manifest: {
        id: process.env.ALPHA ? '/app/' : process.env.BETA ? '/appbeta/' : '/app/',
        name:  process.env.ALPHA ? 'my app alpha' : process.env.BETA ? 'my app beta' : 'app',
        [...etc, etc]
      }
    },
```
but in the vite world, manifest is a separate file, and these items break the build process. So I assumed I would need to use something like `extendManifestJson (json)`, but I can't figure out how to pass items that are env dependant (like the above) into my manifest.  is there some guide to how this command works and adds items to the manifest?

**Comment by rstoenescu** [comment, marked as answer]
```
extendManifestJson (json) {
   json.id = process.env.ALPHA ? '/app/' : process.env.BETA ? '/appbeta/' : '/app/'
}
```
